apps/web-giddh/src/assets/css/style-2.scss

```diff
@@ -8496,6 +8496,9 @@ body.update-scroll-hidden {
 .manage-account-popup.full-screen-modal{
     z-index: 999991;
 }
+.v-align-top {
```

Comment:
there is a predefined class="align-top" for vertical-align: top; Please add the class wherever required and remove this from here

Reply:
align-top is for td only, i have added my class name with comma with this.